### PR TITLE
Update Link Styling

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/styles/class-overrides.scss [hunk 1]
```diff
@@ -44,11 +44,13 @@
...
 @mixin link-size {
-  border-bottom-width: units('1px') * 3;
+  // border-bottom-width: units('1px') * 3;
+  border-bottom-width: units('05');
 }
 
 @mixin link-size-expanded {
-  border-bottom-width: units('1px') * 10;
+  // border-bottom-width: units('1px') * 10;
+  border-bottom-width: units('105');
```
Comment (saifulislamdev) [inline src/styles/class-overrides.scss:53]: Thanks for solving the conflicts! I took this directly from the Figma, and there were no conflicts in the branch I was working in, so I didn't have to address these.

File: src/styles/class-overrides.scss [hunk 2]
```diff
@@ -116,11 +122,15 @@
   border-bottom-style: solid;
   border-bottom-color: color('primary');
   &:hover {
+    text-decoration: none;
```
Comment (saifulislamdev) [inline src/styles/class-overrides.scss:125]: Is this necessary if it's already in line 120?

**Review by saifulislamdev — APPROVED** [review]

**Review by saifulislamdev — CHANGES_REQUESTED** [review]
Only one minor change, if needed.

File: src/styles/class-overrides.scss (hunk 1 above)
Comment (aannyyaa) [reply src/styles/class-overrides.scss:53]: These weren't conflicts - I was just trying to use the USWDS units. So they are slightly different sizes than in the figma

File: src/styles/class-overrides.scss (hunk 2 above)
Comment (aannyyaa) [reply src/styles/class-overrides.scss:125]: Without this line, the white underline still showed up on hover in addition to the bottom border

File: src/styles/class-overrides.scss (hunk 1 above)
Comment (saifulislamdev) [reply src/styles/class-overrides.scss:53]: I was referring to the overlap you mentioned earlier.

**Review by saifulislamdev — APPROVED** [review]

**Pull request merged**